src/gausskernel/storage/access/k2/pg_gate_api.cpp

```diff
@@ -322,8 +321,8 @@ K2PgStatus PgGate_ExecCreateTable(const char *database_name,
                               bool if_not_exist,
                               bool add_primary_key,
                               const std::vector<K2PGColumnDef>& columns) {
-    elog(LOG, "PgGateAPI: PgGate_NewCreateTable %s, %s, %s", database_name, schema_name, table_name);
-    auto [status, is_pg_catalog_table, schema] = makeSChema(schema_name, columns);
+    elog(DEBUG5, "PgGateAPI: PgGate_NewCreateTable %s, %s, %s", database_name, schema_name, table_name);
```

Comment:
we might want to keep the log level as LOG for now to make the debug easier. DEBUG5 is way too verbose to enable in opengauss. We could fine tune the log level after we are done with bug bashing.